src/plugins/notebook/components/Notebook.vue

```diff
@@ -345,6 +374,43 @@ export default {
             this.removeDefaultClass(this.domainObject.identifier);
             clearDefaultNotebook();
         },
+        lockPage() {
+            let prompt = this.openmct.overlays.dialog({
+                iconClass: 'alert',
+                message: "This action will lock this page and disallow any new entries, or editing of existing entries. Do you want to continue?",
+                buttons: [
+                    {
+                        label: 'Lock Page',
+                        callback: () => {
+                            let sections = this.getSections();
+                            this.selectedPage.isLocked = true;
+
+                            // cant be default if it's locked
+                            if (this.selectedPage.id === this.defaultPageId) {
```

Comment:
When you take a new snapshot of a view and save it to a notebook, that will make it a default notebook - Have we covered the cases where that notebook might be locked?

Reply:
We should be covered here, you only have the option to save it to the default notebook (if one exists) otherwise it just gets saved into the snapshots holder.